pkg/scheduler/metrics/metrics.go

```diff
@@ -230,6 +232,20 @@ func InitMetrics() {
 			Help:           "Number of running goroutines split by the work they do such as binding.",
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"operation"})
+	BatchUsageStats = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      SchedulerSubsystem,
+			Name:           "batch_use_results",
+			Help:           "Counts of results when we attempt to use batching.",
+			StabilityLevel: metrics.ALPHA,
+		}, []string{"result"})
+	BatchEventStats = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      SchedulerSubsystem,
+			Name:           "batch_event_stats",
+			Help:           "Counts of various interesting events during batching.",
+			StabilityLevel: metrics.ALPHA,
+		}, []string{"event", "reason"})
```

Comment:
Let's rename them to `foobar_total`. That is how typically we name counter metrics:
https://github.com/kubernetes/kubernetes/blob/master/test/instrumentation/testdata/stable-metrics-list.yaml